src/test/ui/typeck/struct-enum-wrong-args.stderr

```diff
@@ -29,6 +29,12 @@ LL |     let _ = Wrapper();
    |             ^^^^^^^-- supplied 0 arguments
    |             |
    |             expected 1 argument
+   |
+note: tuple struct defined here
+  --> $DIR/struct-enum-wrong-args.rs:2:8
+   |
+LL | struct Wrapper(i32);
+   |        ^^^^^^^
```

Comment:
I think this changed because I fixed `def_ident_span`...? I guess it's a nice unexpected improvement :)

Reply:
Drive by fixes are the best fixes.